### DISCUSSION TITLE
Tutorial:1 add command for windows users.

### DISCUSSION THREAD

**Discussion by Rchanis** [encode/django-rest-framework#9098] [Ideas & Suggestions]
In [Setting up a new evironment](https://github.com/encode/django-rest-framework/blob/580bf45ccfd5c423a938729907d813f4862dca38/docs/tutorial/1-serialization.md#setting-up-a-new-environment)
The command:
`source env/bin/activate`
It will not work for a Windows user; Windows users should use
`.\env\Scripts\activate`

**Comment by auvipy** [comment]
yeah that is legit!
